Headers/Foundation/NSKeyedArchiver.h

```diff
@@ -87,6 +87,16 @@ extern "C" {
  */
 + (NSData*) archivedDataWithRootObject: (id)anObject;
 
+  
+/**
```

Comment:
You should add guards for MacOS 10.13 around this method.

Reply:
Done